lua/nvim-treesitter/parsers.lua

```diff
@@ -1562,6 +1563,15 @@ list.pony = {
   maintainers = { "@amaanq", "@mfelsche" },
 }
 
+list.powershell = {
+  install_info = {
+    url = "https://github.com/airbus-cert/tree-sitter-powershell",
+    files = { "src/parser.c", "src/scanner.c" },
+    branch = "main",
+  },
+  maintainers = { "citronneur" },
```

Comment:
This refers to the _query_ maintainer (i.e., who gets pinged if a parser update breaks our query and -- in a timely fashion -- makes the necessary changes here). That person should have volunteered for this position.

Reply:
Thanks for clarification